tools/pypy-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+apt-get -yq update
+apt-get -yq install libatlas-dev libatlas-base-dev liblapack-dev
+wget http://buildbot.pypy.org/nightly/py3.6/pypy-c-jit-latest-linux64.tar.bz2 -O pypy.tar.bz2
```

Comment:
Nightlies are our best bet for stability at the moment?

Reply:
NumPy uses many of the non-[`Py_LIMITED_API`](https://docs.python.org/3/c-api/stable.html) CPython functions. Using the nightlies will ensure NumPy's build can be fixed for PRs that expose unimplemented API functions 24 hours after those functions are implemented in PyPy.

Reply:
The nightlies are quite stable. Most breakage modes will prevent a broken nightly from being uploaded. 

Since NumPy uses the [`Py_LIMITED_API'](https://docs.python.org/3/c-api/stable.html) which PyPy has only partially implemented, a more likely scenario is that NumPy will start using a C API function that PyPy has not yet implemented. CI will fail until PyPy catches up. By using nightlies the new NumPy PR will not have to wait for a PyPy release.

Reply:
Ok, sounds like you've thought about this carefully, thanks Matti.